CHANGELOG.md

```diff
@@ -6,6 +6,35 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.0-rc.1] - 2023-08-10
+
+> How, dear sir, did you cross the flood? By not stopping, friend, and by not
+> straining I crossed the flood
+
+This new runc release includes all the changes from the v1.1 patch releases.
```

Comment:
I think we have to say here up to which point (up to 1.1.8, for now).

Reply:
If we do a 1.2, we will stop doing 1.1.z releases, right? Just like we did with 1.0.z?

Reply:
When we do 1.2.0 we will stop 1.1.x, but not when we do 1.2.0-rc.1 I guess. So I've added it includes all the fixes until v1.1.9.

Thanks!